### PR TITLE
Add missing copr and builddep plugins documentation

### PR DESCRIPTION
**Pull request by m-blaha**

### CODE REVIEW

File: doc/dnf5_plugins/builddep.8.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+``dnf5 builddep [options] [<package>...]``
+
+
+Description
+===========
+
+Install missing dependencies for building an RPM package.
+
+.. warning:: Build dependencies in a source package (i.e. src.rpm) might be different
+             than you would expect because they were evaluated according macros
```
Comment (inknos) [inline doc/dnf5_plugins/builddep.8.rst:37]: I would replace "they" with "source packages". more clear

File: doc/dnf5_plugins/builddep.8.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+=======
+
+``--allowerasing``
+    Allow erasing of installed packages to resolve dependencies resolution problems.
+
+``--skip-unavailable``
+    Allow skipping build dependencies not available in repositories. All available build dependencies will be installed.
+
+``-D "macro expr", --define="macro expr"``
+    Define the RPM macro named "macro" to the value "expr" when parsing spec files. Does not apply for source rpm files.
```
Comment (inknos) [inline doc/dnf5_plugins/builddep.8.rst:50]: I would rephrase this like 'Define a rpm macro. Set the value "expr" to the macro "macro" when parsing spec files. ...'
or something a written in a shorter form than what's in the committed file. what do you think?

File: doc/dnf5_plugins/repoclosure.8.rst [hunk 3]
```diff
@@ -22,14 +22,14 @@
...
 Synopsis
 ========
 
 ``dnf5 repoclosure [options] [<pkg-spec>...]``
 
 
+Description
+===========
 
 The ``repoclosure`` command in DNF5 allows you to analyze package metadata from multiple repositories. It checks all dependencies of the packages and provides a list of packages that have unresolved dependencies.
```
Comment (inknos) [inline doc/dnf5_plugins/repoclosure.8.rst:34]: What about
"The ``repoclosure`` command allows you to analyze package metadata from multiple repositories. It checks all dependencies of the packages and provides a list of packages that have unresolved dependencies."

File: doc/dnf5_plugins/repoclosure.8.rst [hunk 4]
```diff
@@ -57,22 +57,17 @@ Options
...
-    dnf5 repoclosure --repo rawhide --arch noarch --arch x86_64
+========
 
-Display list of unresolved dependencies for zmap package from rawhide repository::
+``dnf5 repoclosure``
+    Display list of unresolved dependencies for all enabled repositories.
 
-    dnf5 repoclosure --repo rawhide zmap
+``dnf5 repoclosure --repo rawhide --arch noarch --arch x86_64``
+    Display list of unresolved dependencies for rawhide repository and packages with architecture noarch and x86_64.
```
Comment (inknos) [inline doc/dnf5_plugins/repoclosure.8.rst:67]: I believe this is more correct.

Display *a* list of unresolved dependencies...

See below too

**Review by inknos — CHANGES_REQUESTED** [review]

File: doc/dnf5_plugins/builddep.8.rst (hunk 1 above)
Comment (m-blaha) [reply doc/dnf5_plugins/builddep.8.rst]: It refers to dependencies, actually. But I've replaced the pronoun anyway.

File: doc/dnf5_plugins/builddep.8.rst (hunk 2 above)
Comment (m-blaha) [reply doc/dnf5_plugins/builddep.8.rst]: Changed.

File: doc/dnf5_plugins/repoclosure.8.rst (hunk 3 above)
Comment (m-blaha) [reply doc/dnf5_plugins/repoclosure.8.rst]: Changed.

File: doc/dnf5_plugins/repoclosure.8.rst (hunk 4 above)
Comment (m-blaha) [reply doc/dnf5_plugins/repoclosure.8.rst]: Thanks, fixed.

**Comment by inknos** [comment]
lgtm thanks
